src/ert/_c_wrappers/enkf/analysis_config.py

```diff
@@ -173,26 +156,29 @@ def get_max_runtime(self) -> int:
     def set_max_runtime(self, max_runtime: int):
         self._max_runtime = max_runtime
 
-    def free(self):
-        self._free()
+    def active_module_name(self) -> str:
```

Comment:
we have a bit of inconsistency as to when we use the `get` verb at the start of a method, but i don't feel very strongly about it and don't know which way i would advocate, in general, or in this case...

Reply:
makes sense, thanks for explaining!